sklearn-example/digits.py

```diff
@@ -0,0 +1,46 @@
+import struct
+from enum import IntEnum
```

Comment:
Do we need to import `IntEnum`, `re`, and `json`? It doesn't look like they're used anywhere.